lib/proxy-protocol.regexp.js

```diff
@@ -80,7 +80,7 @@ The maximum line lengths the receiver must support including the CRLF are :
 
 var IPv4 = '(((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?).){3}(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?))',
     IPv6 = '([0-9a-fA-F]{0,4}:){2,7}(:|[0-9a-fA-F]{1,4})',
-    Port = '([1-9][0-9]{0,3}|[1-5][0-9]{4}|6[0-4][0-9]{3}|65[0-2][0-9]{2}|653[0-2][0-9]|6533[0-5])'; // 1..65335
+    Port = '([1-9][0-9]{0,3}|[1-5][0-9]{4}|6[0-4][0-9]{3}|65[0-4][0-9]{2}|655[0-3][0-9]|6553[0-5])'; // 1..65535
```

Comment:
@mvalo range regexp for ports recreated on an online web tool

Seems that `... 655[0-3][0-9]|6553[0-5]` was allowing `65539` which is invalid.

Could you please change this to:
```js
Port = '([1-9][0-9]{0,3}|[1-5][0-9]{4}|6[0-4][0-9]{3}|65[0-4][0-9]{2}|655[0-2][0-9]|6553[0-5])'; // 1..65535 @ http://utilitymill.com/utility/Regex_For_Range
```